release_notes/ocp-4-13-release-notes.adoc

```diff
@@ -87,6 +87,14 @@ With {product-title} {product-version}, running `oc describe` on an image now re
 [id="ocp-4-13-networking"]
 === Networking
 
+[id="ocp-4-13-nw-metallb-ipaddresspool-assignment"]
+==== Assign IP addresses to services or namespaces by using MetalLB
+
+With this update, you can assign IP addresses from a MetalLB `IPAddressPool` resource to specific services and namespaces. This is useful in a muti-tenant, bare-metal environment that requires MetalLB to pin IP addresses from an IP address pool to services and namespaces. You can assign services and namespaces to IP addresses from many IP address pools. You can then define the prioritization for these IP address pools so that MetalLB assigns IP addresses from the higher priority IP address pool first.
```

Comment:
> You can then define the prioritization for these IP address pools so that MetalLB assigns IP addresses from the higher priority IP address pool first.

Nit: can we reword like this ? You can then define the prioritization for these IP address pools so that MetalLB assigns IP addresses starting from the higher priority IP address pool.